CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ Changes are grouped as follows
 - Upserts for all resource types
 - Separate read/write fields on data classes
 
+## [Unreleased]
+
+### Changed
```

Comment:
No need to put this in changelog. Documentation is updated without making releases.

Reply:
That's perfect!